M2/Macaulay2/m2/methods.m2

```diff
@@ -700,15 +700,24 @@ Function Thing = (f,x,e) -> (
      if not storefuns#?f then error("no method for storing values of function ", toString f);
      storefuns#f (x,e))
 
--- defined in d/actors4.d
-locate' = locate -- TODO: why does (net, FunctionBody) in nets.m2 need locate'?
-locate = method(Dispatch => Thing, TypicalValue => Sequence)
-locate Nothing    := Sequence => x -> locate' x
-locate Function   := Sequence => x -> locate' x
-locate Pseudocode := Sequence => x -> locate' x
-locate Sequence   := Sequence => x -> locate' x
-locate Symbol     := Sequence => x -> locate' x
-locate List       := List     => x -> apply(x, locate)
+FilePosition = new Type of BasicList
```

Comment:
Would it make sense to move both `FilePosition` and `locate` to `debugging.m2`?  They're not used until there, and also various other locating functions are defined there.

Reply:
I can try, but I'm a little confused with all the complicated dependencies between files.

Reply:
I checked before commenting. Here's a little hack I use for searching the first usage of a symbol roughly by order of files being loaded:
```bash
reset ; for src in startup.m2 Core.m2 `cat loadsequence`; do grep -iHn "\bFilePosition\b" $src; done
```